spec/support/saml_auth_helper.rb

```diff
@@ -151,7 +155,7 @@ def saml_request
   end
 
   def key
-    @key ||= Rails.application.secrets.saml_client_private_key
+    @key ||= File.read("#{Rails.root}/config/saml_client_private_key.enc")
```

Comment:
Be consistent about including _private in the .env filensame?  Seems the other key is simply called saml.key.enc.

Perhaps this method should be called client_private_key or match the private_key method name above?

Not stickily related to this change though, so ignore me if you like.